src/decaylanguage/modeling/decay.py

```diff
@@ -121,4 +121,9 @@ def _make_graphviz(self):
             return d
 
         def _repr_svg_(self):
-            return self._make_graphviz()._repr_svg_()
+            try:
+                return (
+                    self._make_graphviz()._repr_image_svg_xml()
+                )  # for graphviz >= 0.19
+            except AttributeError:
+                return self._make_graphviz()._repr_svg_()  # for graphviz < 0.19
```

Comment:
Same thing here! Otherwise looks good.

Reply:
Ah, last night I somehow forgot to push the changes here! Done now.